<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SizeSelectionWidget</class>
 <widget class="QWidget" name="SizeSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="sizeGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Pixel Resolution</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="0" column="1">
       <widget class="QComboBox" name="presetsSizeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>14</number>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>Custom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">2 x 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">8 x 8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">16 x 16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">32 x 32</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">64 x 64</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">128 x 128</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">256 x 256</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">160 x 120</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">320 x 240  (QVGA)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">640 x 480  (VGA)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">720 x 480  (NTSC)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">768 x 576  (PAL)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">800 x 600  (SVGA)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">1024 x 768 (XGA)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">1280 x 720 (HD)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">1600 x 1200 (UXGA)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">1920 x 1080 (FHD)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2048 x 1536 (QXGA)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2048 x 1152 (QWXGA)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Width x Height </string>
        </property>
        <property name="buddy">
         <cstring>presetsSizeComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QWidget" name="sizeGrid" native="true">
        <layout class="QGridLayout" name="sizeGridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="w">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Width</string>
           </property>
           <property name="buddy">
            <cstring>widthSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="widthSpinBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>3840</number>
           </property>
           <property name="singleStep">
            <number>64</number>
           </property>
           <property name="value">
            <number>1024</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="heightSpinBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>2160</number>
           </property>
           <property name="singleStep">
            <number>64</number>
           </property>
           <property name="value">
            <number>768</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="h">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Height</string>
           </property>
           <property name="buddy">
            <cstring>heightSpinBox</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>presetsSizeComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>SizeSelectionWidget</receiver>
   <slot>setSizePreset(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>sizeChanged()</signal>
  <signal>widthChanged(int)</signal>
  <signal>heightChanged(int)</signal>
  <slot>setSizePreset(int)</slot>
 </slots>
</ui>
